app/routes/core.rb

```diff
@@ -21,7 +21,7 @@ class Core < Sinatra::Application
         enable :show_exceptions
       end
 
-      unless settings.test?
+      unless settings.test? || settings.development?
```

Comment:
The commit that originally added it was here: https://github.com/exercism/exercism.io/commit/39a86ee4637e0547c208cb47561256daf3445dce

And that should also be handled by the [configure block above.](https://github.com/exercism/exercism.io/blob/39a86ee4637e0547c208cb47561256daf3445dce/app/routes/core.rb#L17)

Is there something strange about @kytrinyx's setup that is causing the `enable :show_exceptions` to not work the same way that it does for us?